### PR TITLE
PICT-235: Configure due date period for bank transfer payments

### PR DESCRIPTION
**Pull request by NghiaDTr**

### CODE REVIEW

File: processor/src/utils/map.utils.ts [hunk 1]
```diff
@@ -71,8 +72,7 @@ const getSpecificPaymentParams = (method: PaymentMethod | CustomPaymentMethod, p
       return { applePayPaymentToken: paymentRequest.applePayPaymentToken ?? '' };
     case PaymentMethod.banktransfer:
       return {
-        dueDate: paymentRequest.dueDate ?? '',
-        billingEmail: paymentRequest.billingEmail ?? '',
+        dueDate: calculateDueDate(readConfiguration().mollie.bankTransferDueDate),
```
Comment (Tung-Huynh-Shopmacher) [inline processor/src/utils/map.utils.ts:75]: billingEmail is mandatory for bank transfer

**Review by Tung-Huynh-Shopmacher — CHANGES_REQUESTED** [review]

File: processor/src/utils/map.utils.ts (hunk 1 above)
Comment (NghiaDTr) [reply processor/src/utils/map.utils.ts:75]: Yes, but we have to use billingAddress.email instead, according to this: https://docs.mollie.com/reference/extra-payment-parameters#billingemail-string

Comment (Tung-Huynh-Shopmacher) [reply processor/src/utils/map.utils.ts:75]: Did we add the validation for it em?

Comment (NghiaDTr) [reply processor/src/utils/map.utils.ts:75]: @Tung-Huynh-Shopmacher  absolutely yes anh: https://github.com/mollie/commercetools-connector/pull/38/files#diff-4de1da08646e725b1d436cd3c4c667e9adcddc5697dda83bd3d3e2c5951230c0R267

**Review by Tung-Huynh-Shopmacher — APPROVED** [review]

**Pull request merged**